### ISSUE TITLE
Don't require separate /boot partition for systems with both MBR and GPT disks if installing on MBR disk

### ISSUE THREAD

**Issue by IevgenSobko** [Antergos/Cnchi#790]
### Problem:
I have 3 disks on my system.
- SSD with MBR
- SSD with GPT
- Hard drive with GPT
When I installing on first SSD with MBR installer still requires separate /boot partition to be specified.
Only if I disconnect both disks with GPT I'm able to install system without separate /boot partition.

**Comment by karasu** [comment]
I suppose you did boot in UEFI mode, right?
That's why Cnchi asks for a /boot partition, as it thinks that it is needed.
Let us think about this, maybe we could add the possibility to bypass this requirement by showing some kind of warning message.

**Comment by IevgenSobko** [comment]
No, I booted in legacy mode, Not UEFI.
I think the problem is that installer checks all the disks available and if some of them are with GPT it forces a user to specify /boot partition.
When I disconnected both disks with GPT I installed Antergos without a /boot partition.
I believe it is related to GPT not to UEFI.
I had the same behavior with my Dell XPS laptop that shipped by default with GPT.
I couldn't install without /boot. But after format disk as MBR I installed without separate /boot.

**Comment by Dizirgee** [comment]
This was probably changed in meantime.
Yesterday I installed from latest iso 17.12. , booted in UEFI , 5 disks (3x GPT HDD, 1x GPT SSD, 1x MBR SSD)  used gpt ssd at sda for /
and while manually assigning partitions I seen red circle for /boot "button" but I ignored it and didn't assign it and I could go to next step and installation finished.

Though at first I was at pause and trying to figure out if/what /boot partition should I use as I didn't even try to continue installation due to that red circle (assumed it wouldn't allow me), but on second try I just tried to click next and it worked :)
